app/views/api/v1/release_packages/pypi/simple/index.html.haml

```diff
@@ -7,6 +7,6 @@
     %h1 Links for #{product.name} by #{account.name}
 
     - artifacts.each do |artifact|
-      - url  = url_for([:v1, account, artifact, anchor: checksum_for(artifact, delimiter: '=')])
+      - url = url_for([:v1, account, artifact, anchor: checksum_for(artifact, delimiter: '='), protocol: 'https'])
```

Comment:
I don't understand why the default URL options in `config/application.rb` are not applying here...